articles/logic-apps/error-exception-handling.md

```diff
@@ -334,6 +268,37 @@ For limits on scopes, see [Limits and config](logic-apps-limits-and-config.md).
 
 <a name="get-results-from-failures"></a>
 
+### Use "scope" in the designer
+
+#### Step 1: Open the Logic App Designer
```

Comment:
We're also linking to a generic doc about how to add triggers and actions to centralize these steps due to frequent UX updates. Here's the steps rewritten per our guidelines.

```suggestion
1. In the [Azure portal](https://portal.azure.com), open your logic app workflow in the designer.

    Your workflow must already have a trigger that starts the workflow.

1. On the designer, [follow these generic steps to add a **Control** action named **Scope** to your workflow](/azure/logic-apps/create-workflow-with-trigger-or-action#add-action).

1. In the **Scope** action, [follow these generic steps to the add actions to run](/azure/logic-apps/logic-apps-control-flow-run-steps-group-scopes#add-steps-to-scope), for example:

   :::image type="content" source="media/error-exception-handling/add-actions-into-scope.png" alt-text="Screenshot shows workflow designer with actions grouped inside the scope.":::

   The following list shows some example actions that you might include inside a **Scope** action:
   
   - Get data from an API.
   - Process the data.
   - Save the data to a database.

1. Now define the "run after" rules for running the actions in the scope.

   1. On the designer, select the **Scope** title. When the scope's information pane opens, select **Settings**.

   1. If you have more than one preceding action in the workflow, from the **Select actions** list, select the action after which you want to run the scoped actions.

   1. For the selected action, select all the action statuses that can run the scoped actions.

      In other words, any of the chosen statuses that result from the selected action cause the actions in the scope to run.

   In the following example, the scoped actions run after the **HTTP** action completes with any of the selected statuses:

   :::image type="content" source="media/error-exception-handling/set-run-after-in-scope.png" alt-text="Screenshot shows scope action's Settings tab, run after section, and selected action statuses that run the scoped actions.":::

```